### PR TITLE
Add arabic and hungarian and update old translations

### PR DESCRIPTION
**Pull request by daniserejo**
Add new translations for arabic and hungarian and update old translations

### CODE REVIEW

**Review by igorbrasileiro — CHANGES_REQUESTED** [review]
Can you add the changelog entry?

**Comment by daniserejo** [comment]
> 
> 
> Can you add the changelog entry?

@igorbrasileiro For now this PR is for a future project, so we'll maintain it as a draft until we finish it :)

**Review by camilavcoutinho — COMMENTED** [review]
@sheilagomes don't rename dannish, finnish, japanese, korean and romanian files. There is no need 😄

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,14 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
 ## [Unreleased]
 
+### Added
+
+- I18n Ar, Hu.
+
+### Fixed
+
+- I18n Da, Fi, Hu, Ja, Ko, and Ro.
+
```
Comment (camilavcoutinho) [inline CHANGELOG.md]:
```suggestion
```

**Review by camilavcoutinho — CHANGES_REQUESTED** [review]

**Review by camilavcoutinho — APPROVED** [review]

**Review by icazevedo — APPROVED** [review]

**Review by igorbrasileiro — APPROVED** [review]

**Pull request merged**